org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/MavenProjectImporter.java

```diff
@@ -232,8 +234,14 @@ public void importToWorkspace(IProgressMonitor monitor) throws CoreException, Op
 	}
 
 	private long getLastWorkspaceStateModified() {
-		File workspaceStateFile = MavenPluginActivator.getDefault().getMavenProjectManager().getWorkspaceStateFile();
-		return workspaceStateFile.lastModified();
+		Bundle bundle = Platform.getBundle(IMavenConstants.PLUGIN_ID);
```

Comment:
`FrameworkUtil.getBundle(IMaven.class)` should be a bit quicker to obtain the `o.e.m2e.core` bundle.